tutorials/Getting_Started/README.md

```diff
@@ -30,6 +30,17 @@ packaging JDK 14 or later must be used.
 
 ![Create new project 3](screen5.png)
 
+### Update the wizard plugin
+The compose plugin version used in the wizard above may be out of date. Update the version of the plugin to the latest available. 
+E.g. at time of writing that was kotlin 1.4.30 and compose plugin version 0.3.0-build152.
+Find and update this in the build.gradle.kts file.
```

Comment:
Please provide link to https://github.com/JetBrains/compose-jb/tags to discover most recent version.

Reply:
Updated with links in last commit.